docs/reference/facemesh.md

```diff
@@ -0,0 +1,175 @@
+# Facemesh
+
+
+<center>
+    <img style="display:block; max-height:20rem" alt="A screenshot of a video video feed where a person sits at their chair inside of a bedroom while green dots are drawn over different locations on their face." src="_media/reference__header-facemesh.jpg">
+</center>
+
+
+## Description
+
+Facemesh is a machine-learning model that allows for facial landmark detection in the browser. It can detect multiple faces at once and provides 486 3D facial landmarks that describe the geometry of each face. Facemesh works best when the faces in view take up a large percentage of the image or video frame and it may struggle with small/distant faces.
+
+The ml5.js Facemesh model is ported from the [TensorFlow.js Handpose implementation](https://github.com/tensorflow/tfjs-models/tree/master/facemesh#keypoints).
+
+## Quickstart
+
+```js
+let predictions = [];
+const video = document.getElementById('video');
+
+// Create a new facemesh method
+const facemesh = ml5.facemesh(video, modelLoaded);
+
+// When the model is loaded
+function modelLoaded() {
+  console.log('Model Loaded!');
+}
+
+// Listen to new 'predict' events
+facemesh.on('predict', results => {
+  predictions = results;
+});
+```
+
+
+## Usage
+
+### Initialize
+You can initialize ml5.facemesh with an optional `video`, configuration `options` object, or a `callback` function.
+```js
+const poseNet = ml5.facemesh(?video, ?options, ?callback);
+```
+
+#### Parameters
+* **video**: OPTIONAL. Optional HTMLVideoElement input to run predictions on.
+* **options**: OPTIONAL. A object that contains properties that effect the Facemesh model accuracy, results, etc. See documentation on the available options in [TensorFlow's Facemesh documentation](https://github.com/tensorflow/tfjs-models/tree/master/facemesh#parameters-for-facemeshload).
```

Comment:
Probably best to list out all of the options here instead, since I added flipHorizontal as an option.

Reply:
Yes! I'd def go for listing them like:

```js
const options = {
  flipHorizontal: (true | false)
  ...
}
```